blackbox/cli.py

```diff
@@ -77,11 +86,10 @@ def run() -> bool:
 @click.option('--version', is_flag=True, help="Show version and exit")
 def cli(config, init, version):
     """
-    BLACKBOX
+    BLACKBOX.
 
     Backup database to external storage system
     """
```

Comment:
We probably shouldn't put punctuation here, because this is used by `click` to output a sort of headline for the help command.

Instead, we can put a linter exception for this docstring by using `#noqa`

Reply:
You do need to do the `# noqa` thing here after this docstring, or it will not pass the linter.